### PR TITLE
docs(Parser): allow to pass custom worker syntax

### LINKED ISSUE #3999: Document webpack change: allow to pass custom worker syntax
*A pull request by @sokra was merged and maintainers requested a documentation change.*

See pull request: https://github.com/webpack/webpack/pull/11531

---

**What kind of change does this PR introduce?**
feature

**Did you add tests for your changes?**
yes

**Does this PR introduce a breaking change?**
no

**What needs to be documented once your changes are merged?**
for javascript files one can customize WebWorker handling syntax with `parser.worker`.

Example to support the `web-worker` package:

```js
	module: {
		rules: [
			{
				test: /\.m?js$/,
				parser: {
					worker: ["default from web-worker", "..."]
				}
			}
		]
	}
```

### PR DESCRIPTION
**Pull request by pranshuchittora**
Closes #3999

### CODE REVIEW

File: src/content/configuration/module.mdx
Comment (EugeneHlushko) [inline src/content/configuration/module.mdx:301]: Why is the "..." required?

Comment (pranshuchittora) [reply src/content/configuration/module.mdx:301]: From tests 
https://github.com/webpack/webpack/blob/28580af775de8e166aed23189988199d4aa0c42e/test/configCases/worker/custom-worker/webpack.config.js#L11

Comment (EugeneHlushko) [reply src/content/configuration/module.mdx:301]: Yes i saw that PR, i dont understand what it does, this doc is cryptic to average reader if maintainers have no idea... I suggest we explain the `"..."` by asking relevant code contributors e.g. @sokra in slack before proceeding with the merge

Comment (pranshuchittora) [reply src/content/configuration/module.mdx]: Thanks. I have added the following info about the `"..."`. 

https://github.com/webpack/webpack/pull/11531#issuecomment-700529559

Comment (EugeneHlushko) [reply src/content/configuration/module.mdx]: 👍

**Review by EugeneHlushko — APPROVED** [review]

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**
